<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBeatsPicoammeter</class>
 <widget class="QMainWindow" name="FormBeatsPicoammeter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beats Picoammeter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QELabel" name="lblCH2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-DI-PA:getCurrent2</string>
    </property>
    <property name="precision">
     <number>10</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
    <property name="notation" stdset="0">
     <enum>QE::Scientific</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>411</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QELabel" name="lblCH4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-DI-PA:getCurrent4</string>
    </property>
    <property name="precision">
     <number>10</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
    <property name="notation" stdset="0">
     <enum>QE::Scientific</enum>
    </property>
   </widget>
   <widget class="QELabel" name="lblCH3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-DI-PA:getCurrent3</string>
    </property>
    <property name="precision">
     <number>10</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
    <property name="notation" stdset="0">
     <enum>QE::Scientific</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QELabel" name="lblCH1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-DI-PA:getCurrent1</string>
    </property>
    <property name="precision">
     <number>10</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
    <property name="format" stdset="0">
     <enum>QE::Default</enum>
    </property>
    <property name="notation" stdset="0">
     <enum>QE::Scientific</enum>
    </property>
    <property name="updateOption" stdset="0">
     <enum>QELabel::Text</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>XBPM Blade Top</string>
    </property>
   </widget>
   <widget class="QELabel" name="inReadback_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-CO-XPBM-BLADETOP-THC:getTemp</string>
    </property>
    <property name="precision">
     <number>2</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QELabel" name="inReadback_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-CO-XPBM-BLADEBOT-THC:getTemp</string>
    </property>
    <property name="precision">
     <number>2</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>XBPM Blade Bot</string>
    </property>
   </widget>
   <widget class="QELabel" name="inReadback_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-CO-XPBM-CUMASK-THC:getTemp</string>
    </property>
    <property name="precision">
     <number>2</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>copper mask</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor readback </string>
    </property>
   </widget>
   <widget class="QENumericEdit" name="inPosition">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="variable" stdset="0">
     <string>I10R1-MO-MC2:FE-BPM-STP-TRSY1.VAL</string>
    </property>
    <property name="autoScale">
     <bool>false</bool>
    </property>
    <property name="precision">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-999.999900000000025</double>
    </property>
    <property name="maximum">
     <double>999.999900000000025</double>
    </property>
    <property name="writeOnEnter">
     <bool>false</bool>
    </property>
    <property name="writeOnFinish">
     <bool>false</bool>
    </property>
    <property name="writeOnChange">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QELabel" name="inReadback">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>I10R1-MO-MC2:FE-BPM-STP-TRSY1.RBV</string>
    </property>
    <property name="precision">
     <number>4</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QESimpleShape" name="lowLimitSwitch">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="colour0" stdset="0">
     <color>
      <red>5</red>
      <green>240</green>
      <blue>5</blue>
     </color>
    </property>
    <property name="colour1" stdset="0">
     <color>
      <red>255</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
    <property name="variable" stdset="0">
     <string>I10R1-MO-MC2:FE-BPM-STP-TRSY1.LLS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lowLimitSwitchLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Limit Switch</string>
    </property>
   </widget>
   <widget class="QESimpleShape" name="highLimitSwitch">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="colour0" stdset="0">
     <color>
      <red>5</red>
      <green>240</green>
      <blue>5</blue>
     </color>
    </property>
    <property name="colour1" stdset="0">
     <color>
      <red>255</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
    <property name="variable" stdset="0">
     <string>I10R1-MO-MC2:FE-BPM-STP-TRSY1.HLS</string>
    </property>
   </widget>
   <widget class="QLabel" name="highLimitSwitchLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>High Limit Switch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset</string>
    </property>
   </widget>
   <widget class="QELabel" name="inReadback_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-DI-PA:getPositionY</string>
    </property>
    <property name="useDbPrecision">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QELineEdit" name="qelineedit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-DI-PA:Calculate.H</string>
    </property>
    <property name="displayAlarmStateOption" stdset="0">
     <enum>QE::Never</enum>
    </property>
   </widget>
   <widget class="QELineEdit" name="qelineedit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-DI-PA:Calculate.I</string>
    </property>
    <property name="displayAlarmStateOption" stdset="0">
     <enum>QE::Never</enum>
    </property>
   </widget>
   <widget class="QEComboBox" name="qecombobox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="variable" stdset="0">
     <string>I10FE-DI-PA:Calculate.SCAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculation Rate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEComboBox</class>
   <extends>QComboBox</extends>
   <header>QEComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
